lib/etcdv3.rb

```diff
@@ -17,10 +17,13 @@ class Etcdv3
   def_delegators :@conn, :user, :password, :token, :endpoints, :authenticate
 
   attr_reader :conn, :options
+  attr_reader :credentials, :options
```

Comment:
Looks like you doubled up on :options, may as well just move :credentials into the above attr_reader